bats_test/base-reward/base-reward.bats

```diff
@@ -9,3 +9,8 @@ load '/home/node/packages/node_modules/bats-assert/load.bash'
     run -0 yarn ardrive base-reward
     assert_output --regexp '^[0-9]+$'
 }
+
+@test "'ardrive base-reward --gateway' will error with an invalid Arweave gateway" {
+    run yarn ardrive base-reward --gateway http://fakeway.net
```

Comment:
Should also check exit status. I think there's 2 ways to do that:
https://bats-core.readthedocs.io/en/stable/writing-tests.html#run-test-other-commands
1) Check the value of the $status variable.
2) Invoke `run -1` which checks that the exit status is 1.

I'm not sure what the tradeoffs are of each but the second option looks pretty clean.

Reply:
Same comment for all other examples of this in the PR.

Reply:
Updated here: [033556a](https://github.com/ardriveapp/ardrive-cli/pull/246/commits/033556a9c53f986d9e182af94521e6a908c4aa1e)